packages/playground/cli/src/run-cli.ts

```diff
@@ -394,7 +394,33 @@ export async function parseOptionsAndRunCLI() {
 			],
 		} as RunCLIArgs;
 
-		await runCLI(cliArgs);
+		const cliServer = await runCLI(cliArgs);
+		if (cliServer === undefined) {
```

Comment:
Do we have explicit result modes for every outcome? `undefined` means "success, no server", Server instance means there's a server running, an exception means an error. That sounds fine for the initial launch. Do we have any way of reporting errors further down the road other than the logger? If not, that may be a good exploration after this PR.